plugin-hrm-form/src/states/profile/selectors.ts

```diff
@@ -64,3 +64,12 @@ export const selectProfileRelationshipsByType = (
   profileId: ProfileIdParam,
   type: t.ProfileRelationships,
 ) => selectProfileById(state, profileId)?.[type];
+
+export const selectProfileListState = (state: RootState) => {
+  const profileState = selectProfileState(state);
+  return {
+    data: profileState?.profilesList?.data,
+    error: profileState?.profilesList?.error,
+    loading: profileState?.profilesList?.loading,
+  };
+};
```

Comment:
Probably the same as
```suggestion
export const selectProfileListState = (state: RootState) => {
  const profileState = selectProfileState(state);
  return profileState?.profilesList || {};
};
```
?